utils/driver/jcc-11.5.0.0.jar-LICENSE

```diff
@@ -0,0 +1,53 @@
+LICENSE INFORMATION
```

Comment:
In the past we've added text to a README to say where to get the db2 jar file, but not include the jar itself

Reply:
I added the jar to simplify CI/CD, and to have some guarantees about finding the jar file.